src/bin/deku_node.ml

```diff
@@ -155,26 +119,28 @@ let node folder =
     ~context:node.Node.State.interop_context ~on_operation:(fun operation ->
       Flows.received_tezos_operation (Server.get_state ()) update_state
         operation);
-  let () = Node.Server.start ~initial:node in
-  let _server =
-    App.empty
-    |> App.port (Node.Server.get_port () |> Option.get)
-    |> handle_block_level
-    |> handle_received_block_and_signature
-    |> handle_received_signature
-    |> handle_block_by_hash
-    |> handle_protocol_snapshot
-    |> handle_request_nonce
-    |> handle_register_uri
-    |> handle_receive_user_operation_gossip
-    |> handle_receive_consensus_operation
-    |> handle_withdraw_proof
-    |> handle_ticket_balance
-    |> handle_trusted_validators_membership
-    |> App.start
-    |> Lwt_main.run in
-  let forever, _ = Lwt.wait () in
-  Lwt_main.run forever
+  Node.Server.start ~initial:node;
+  Dream.initialize_log ~level:`Warning ();
+  let port = Node.Server.get_port () |> Option.get in
+  Dream.run ~port
+  @@ Dream.router
+       [
+         handle_received_block_and_signature;
```

Comment:
This is one is present two times, is it normal?

Reply:
Nice catch, fixed on a future commit, no, it's not normal but should not be a problem either.